reportportal/templates/analyzer-statefulset.yaml

```diff
@@ -57,15 +60,28 @@ spec:
         - name: AMQP_EXCHANGE_NAME
           value: "analyzer-default"
         - name: ES_HOSTS
-          value: "{{ .Values.elasticsearch.endpoint }}"
+          value: "{{ .Values.elasticsearch.endpoint }}:{{ .Values.elasticsearch.port }}"
```

Comment:
It breaks existing deployments anyway. E.G. a user has `https://elastic.com:9200` in his `.elasticsearch.endpoint` value, that transforms into invalid URL: `https://elastic.com:9200:` (in case of empty port)

Reply:
I discussed this with the developers, they will bring new variables to this form. It is not a problem, but also I want to make all variables the same form like a RabbitMQ, Postgres.

```yaml
rabbitmq:
  endpoint:
    address: <rabbitmq-release-name>-rabbitmq.default.svc.cluster.local
    port: 5672
    user: rabbitmq
    password:

postgresql:
  endpoint:
    address: <postgresql-release-name>-postgresql.default.svc.cluster.local
    port: 5432
    password:
```